Assets/HoloToolkit/UX/Scripts/UsabilityScaler.cs

```diff
@@ -8,25 +8,21 @@ namespace HoloToolkit.Unity
     /// <summary>
     /// A MonoBehaviour which automatically scales an object for better usability across different devices.
     /// </summary>
-    public class UsabilityScaler : StartAwareBehaviour
+    public class UsabilityScaler : MonoBehaviour
     {
         private Vector3 baseScale;
 
-        protected override void OnEnableAfterStart()
+        private void OnEnable()
         {
-            base.OnEnableAfterStart();
-
             baseScale = transform.localScale;
             float usabilityScaleFactor = UsabilityUtilities.GetUsabilityScaleFactor(Camera.main);
```

Comment:
Should this be CameraCache.main now?

Reply:
Yeah, there seems to be a few `Camera.main` references in the new classes that didn't get updated yet.